testsuite/pytests/test_multimeter.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+#
+# test_multimeter.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+import pytest
+import nest
+
+
+@pytest.fixture
+def _set_resolution():
+    """
+    Set resolution to power of two to avoid rounding issues.
+    """
+
+    nest.ResetKernel()
+    nest.resolution = 2**-3
+
+
+# Obtain all models with non-empty recordables list
+models = (model for model in nest.node_models if nest.GetDefaults(model).get('recordables'))
+
+
+@pytest.mark.parametrize('model', models)
+def test_recordables_are_recorded(_set_resolution, model):
+    """
+    Test that recordables are recorded.
+
+    For each model with recordables, set up minimal simulation
+    recording from all recordables and test that data is provided.
+    I also checks that the recording interval can be set.
+
+    @note This test does not check if the data is meaningful.
```

Comment:
```suggestion
    .. note::
       This test does not check if the data is meaningful.
```
Missed this in my first review. Notes should use the rST directive.